spec/factories/order_details.rb

```diff
@@ -4,17 +4,21 @@
   factory :order_detail do
     quantity { 1 }
     created_by { 0 }
+    order_status { OrderStatus.new_status }
```

Comment:
It's a little weird to me that these weren't here before. My guess is we don't use this factory all that often.